src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -614,6 +614,40 @@ public virtual RedirectResult RedirectPermanent(string url)
         }
 
         /// <summary>
+        /// Creates a <see cref="LocalRedirectResult"/> object that redirects to
+        /// the specified local <paramref name="url"/>.
+        /// </summary>
+        /// <param name="url">The local URL to redirect to.</param>
+        /// <returns>The created <see cref="LocalRedirectResult"/> for the response.</returns>
+        [NonAction]
+        public virtual LocalRedirectResult LocalRedirect(string url)
```

Comment:
Consider changing the parameter name to `localUrl` (throughout everything relevant in this PR).